Makefile

```
@@ -8,27 +8,53 @@ GOARCH_aarch64 = arm64
 GOARCH_arm64 = arm64
 GOARCH ?= $(shell echo "$(GOARCH_$(ARCH))")
 
+VERSION := $(shell git describe --tags --always)
+REVISION := $(shell git rev-parse HEAD)
+PACKAGE := github.com/abiosoft/colima/config
+VERSION_VARIABLES := -X $(PACKAGE).appVersion=$(VERSION) -X $(PACKAGE).revision=$(REVISION)
+
+OUTPUT_DIR := _output/binaries
+OUTPUT_BIN := colima-$(OS)-$(ARCH)
+INSTALL_DIR := /usr/local/bin
+BIN_NAME := colima
+
+LDFLAGS := $(VERSION_VARIABLES)
+
+.PHONY: all
 all: build 
 
+.PHONY: clean
 clean:
 	rm -rf _output _build
 
+.PHONY: gopath
 gopath:
 	go get -v ./cmd/colima
 
+.PHONY: fmt
 fmt:
 	go fmt ./...
 	goimports -w .
 
+.PHONY: build
 build:
-	GOOS=$(GOOS) GOARCH=$(GOARCH) OS=$(OS) ARCH=$(ARCH) sh scripts/build.sh
+	go build -ldflags="$(LDFLAGS)" -o $(OUTPUT_DIR)/$(OUTPUT_BIN) ./cmd/colima
```

Seems the variable does not propagate.

```suggestion
	GOOS=$(GOOS) GOARCH=$(GOARCH) go build -ldflags="$(LDFLAGS)" -o $(OUTPUT_DIR)/$(OUTPUT_BIN) ./cmd/colima
```